src/main/resources/metadata.conf

```diff
@@ -15,6 +15,8 @@
 # * display-priority: relative weight, like title vs subtitle. Allowed values: [0, 1, 2]
 # * display-order: numeric value which the UI can use to sort the attributes for display
 # * sufficient-for-query: Boolean, which is for checking if predicate on this field is sufficient to make a query
+# * currency-symbol: used in context of account balances, example: "ꜩ"
+# * currency-symbol-code: used in context of account balances, example: 42793
```

Comment:
Is it possible to add an explanation of what the code should represent, I have no clue, is it a unicode code-point?

Reply:
@anonymoussprocket, could you comment on that? I only know that it's a value which is needed for front-end.